src/qml/CSLOLDialogSettings.qml

```diff
@@ -137,6 +140,31 @@ Dialog {
                     text: qsTr("Enable systray icon")
                     checked: false
                     Layout.fillWidth: true
+
+                    onCheckedChanged: {
+                        if (checked) {
```

Comment:
I'm not sure this is best way to handle this, wouldn't be much simpler to bind to ``enabled``/``visible`` property of system tray itself